platform/README.md

```diff
@@ -3,9 +3,11 @@
 [![Build Status](https://dfe-ssp.visualstudio.com/s198-DfE-Benchmarking-service/_apis/build/status%2FPlatform%20CICD?branchName=main)](https://dfe-ssp.visualstudio.com/s198-DfE-Benchmarking-service/_build/latest?definitionId=2595&branchName=main)
 
 ## Getting started
+[comment]: <> (I think we need a more high level overview of what is in this directory and where to find it.)
```

Comment:
TODO added as introduce is need.

Reply:
Maybe I'm going mad, but I can't see it added in?